src/test/java/com/google/cloud/spanner/pgadapter/ParserTest.java

```diff
@@ -669,16 +670,26 @@ public void testCopyParserWithNamespace() throws Exception {
       assertThat(options.getFormat(), is(equalTo(Format.TEXT)));
     }
 
+    {
```

Comment:
can you modify the test above to be public.\"USERS\" FROM STDIN

Reply:
I added some more tests which explicitly check that case was preserved on the quoted identifiers with namespaces.